### PR TITLE
dwais/dpalazzari/seedFileUpdates

### PR DESCRIPTION
**Pull request by gdwais**
So I lost my consul setup so I'll have to rebuild.  This SHOULD work though - made some notes.  Let me know if you have any questions.  Adding a package so make sure you reinstall modules.

### CODE REVIEW

**Comment by Dpalazzari** [comment]
@gdwais go here for your keys: https://developer.marvel.com/account

**Comment by Dpalazzari** [comment]
@gdwais this doesnt work, it's only an empty array now (characters)

**Comment by gdwais** [comment]
yup - got my consul setup.  working it out now.

**Comment by Dpalazzari** [comment]
One of your comments mentions arrays in arrays, I created a flatten function a couple years ago:

```javascript
module.exports = function flattenArray(arr){
  var finalArray = [].concat.apply([], arr)
  for(i = 0; i < finalArray.length; i++){
    if(finalArray[i].length){
      var finalArray = flattenArray(finalArray)
    }
  }
  return finalArray
}
```

**Comment by gdwais** [comment]
g2g now.  and did it without the extra module!

File: tests/seed.test.js [hunk 1]
```diff
@@ -1,8 +1,18 @@
+const getData = require('../helpers/mapApiData');
+const url = require('../helpers/consulHelper');
+
 test('pulls data from Marvel API to populate db', async () => {
-  const url = require('../helpers/consulHelper');
```
Comment (Dpalazzari) [inline tests/seed.test.js:2]: @gdwais why are you deleting this?

Comment (gdwais) [reply tests/seed.test.js:2]: just moved it out of scope so that you can use it in other tests you might create without have to require it again

Comment (gdwais) [reply tests/seed.test.js:2]: but maybe you don't want it that way - feel free to change it back.  i'll stay in my async lane ;)

Comment (Dpalazzari) [reply tests/seed.test.js:2]: no it's fine I just didnt see it added anywhere else.

**Comment by Dpalazzari** [comment]
Hmmm....still nothing in the array...
Screen Shot 2019-04-28 at 9 26 22 AM

**Comment by gdwais** [comment]
weird - why does the test pass then? I guess you haven't setup an assertion yet?

image

**Comment by Dpalazzari** [comment]
Correct, no assertions yet

**Comment by Dpalazzari** [comment]
I found the fix, closing this Pr.

**Pull request closed without merging**